helm/prometheus-rules/templates/alerting-rules/dns-operator-azure.rules.yml

```diff
@@ -0,0 +1,28 @@
+{{- if (eq .Values.managementCluster.provider.kind "capz") }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels: {{- include "labels.common" . | nindent 4}}
+  name: dns-operator-azure.rules
+  namespace: {{.Values.namespace}}
+spec:
+  groups:
+    - name: dns-operator-azure
+      rules:
+        - alert: ClusterDNSZoneMissing
```

Comment:
should we add an alert for `sum by (method) (rate(error_metric_name{}[5m])) > 0` for 15m ?

Reply:
i already had it and then i dropped it again for some reasons - but yeah, better have it and drop it later ;-)